.github/workflows/check-github-api-limit.yml

```
@@ -0,0 +1,20 @@
+name: Check Github API Rate limit
+
+on:
+  workflow_call:
+
+jobs:
+  check-github-api-rate-limit:
+    runs-on: ubuntu-latest
+    env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+    - name: Check Github API rate limit Unauthorized
+      run: curl https://api.github.com/rate_limit
+    - name: Check Github API rate limit Authorized
+      run: |
+        curl -L \
```

```suggestion
        curl -Ss -L \
```

Silent and show error.